R/visualization_functions.R

```diff
@@ -659,6 +661,7 @@ download_kegg_png <- function(pw_url, f_path, quiet = TRUE) {
   res <- tryCatch({
     cond <- utils::download.file(url = pw_url,
                                  destfile = f_path,
+                                 method = "wb",
```

Comment:
the argument name should be `mode` not `method` (that is for "wget", "curl" etc.).

Reply:
of course :) I don't know how I managed to mixed this up

Reply:
easy mistake, couldn't see it for a while myself :)

CI checks on the way, will merge if successful